### ISSUE TITLE
Avoid using `permutation_test_score` to define chance level

### ISSUE THREAD

**Issue by ArturoAmorQ** [INRIA/scikit-learn-mooc#449]
In [this lecture notebook](https://inria.github.io/scikit-learn-mooc/python_scripts/cross_validation_baseline.html) and [this solution notebook](https://inria.github.io/scikit-learn-mooc/python_scripts/cross_validation_sol_02) we define a notion of _chance level performance_ as a synonym of the `sklearn.model_selection.permutation_test_score`. This has two undesired consequences: 

- Implicitly introduces the concept of `pvalue`. Instead, we can do the shuffling by hand using `sklearn.utils.shuffle` to keep things conceptually simple.
- The concept of _chance level_ depends a lot of context. For the sake of clarity we can refer to this baseline as _permuted score/error_.

**Comment by glemaitre** [comment]
> Implicitly introduces the concept of pvalue. Instead, we can do the shuffling by hand using sklearn.utils.shuffle to keep things conceptually simple.

I don't really see the problem here because we don't use or wither present the p-values. I would really prefer to use the functionality instead of doing the `for` loop and using the `sklearn.utils.shuffle`. I am not convinced that introducing a tool from `sklearn.utils` is a good idea.

> The concept of chance level depends a lot of context. For the sake of clarity we can refer to this baseline as permuted score/error.

I am fine changing the terminology to avoid any misunderstanding of concepts.

**Comment by ogrisel** [comment]
FYI I agree with @ArturoAmorQ and I am working on a PR.
